massa-pool-worker/src/tests/operation_pool_tests.rs

```diff
@@ -62,17 +64,15 @@ fn test_pool() {
             // generate (id, transactions, range of validity) by threads
             let mut thread_tx_lists = vec![Vec::new(); pool_config.thread_count as usize];
 
+            let mut storage = storage_base.clone_without_refs();
```

Comment:
Might be out of scope for this PR, but it might be helpful to think about "why clone_without_refs, and not just clone, in this particular context".

Reply:
Issue made #3605